lucida/template/cpp/Makefile

```diff
@@ -0,0 +1,31 @@
+all:
+	cd server ; \
+	make all ; \
+	cd ../test; \
+	make all ; \
+	cd ..;
```

Comment:
The thrift file is generated inside server and test folder, and `make all` inside server folder include `make thrift`. I have fixed all other issues. Thank you very much!